waspc/data/Generator/templates/server/src/auth/providers/oauth/setupRouter.ts

```diff
@@ -0,0 +1,48 @@
+import { Router } from "express";
```

Comment:
Every provider must have a `setupRouter` method which returns the Express `router`. In this function we are flexibile to do what ever is necessary to make the provider work. 

For oauth providers, we have an endpoint `/login` to get the auth URL, and the `/callback` endpoint which is used to get the actual `access_token` and the user info.

Reply:
Added it